src/sources/postgresql_metrics.rs

```diff
@@ -221,33 +221,59 @@ impl DatnameFilter {
         (null, names.into_iter().collect())
     }
 
+    fn build_match_params(sql: &mut String, len: usize, include: bool) {
+        let op = if include {
+            (" OR", "~")
+        } else {
+            (" AND", "!~")
+        };
+
+        sql.push_str(" WHERE (");
+        for i in 1..=len {
```

Comment:
Does this work if `len == 1` where it'll end up with `WHERE ()`?

Reply:
~It's not possible to get `WHERE ()` because `len` will be at least `1`.~
Ops, it's actually possible if pass empty pattern... will fix :+1: